### ISSUE TITLE
Fix licensing for all sources that will be use for xcol release

### ISSUE THREAD

**Issue by camiplata** [CatalogueOfLife/xcol#26]
1. Overview of the licenses all sources used by the GBIF backbone 
2. Statistics about the licenses ChecklistBank that are beyond cc-0 cc-by
3. Remove unspecified and other  licenses from front end ChecklistBank
4. Remove other licenses on the future

related to #18

**Comment by camiplata** [comment]
Datasets that need to be addressed:

-  [TAXREF](https://www.gbif.org/dataset/0e61f8fe-7d25-4f81-ada7-d970bbb2c6d6) `CC BY NC` - 115618 names
- [Fauna Europaea ](https://www.gbif.org/dataset/90d9e8a6-0ce1-472d-b682-3451095dbc5a)  `CC BY SA`- 45652 names

Work in progress

**Comment by mdoering** [comment]
GRIN: https://npgsweb.ars-grin.gov/gringlobal/taxon/abouttaxonomy

**Comment by camiplata** [comment]
Check again the sources with the last Ymal file, and transfor this information into a google doc so we can keep track of this sources and start a conversaton with the data providers and GBIF

**Comment by camiplata** [comment]
[Google doc](https://docs.google.com/spreadsheets/d/1CU3rR_RC848YnUoe7JpaPfrfRSI3b3-OkVYiAXESjwQ/edit?usp=sharing) including the updated list for the xcol release sources

**Comment by camiplata** [comment]
IUCN licenses has been changed, the GBIF data products is helping with the datasets with unspecified license but most of then are not being maintained and the contact point is outdated.

**Comment by camiplata** [comment]
- [TAXREF](https://www.gbif.org/dataset/0e61f8fe-7d25-4f81-ada7-d970bbb2c6d6) has been contacted by markus
- Thomas is dealing with [Checklist of Danish Fungi](https://www.gbif.org/dataset/f43069fe-38c1-43e3-8293-37583dcf5547), but more importantly he has put up the [Index fungorum api Crawl](https://www.dev.checklistbank.org/dataset/273752/taxon/90154)  with 50.000 + names for fungi 
- [Mammal Species of the World](https://www.gbif.org/dataset/672aca30-f1b5-43d3-8a2b-c1606125fa1b) are joining forces with MDD, bats of the world, as MSWnext so we can take this into account for the normal release next year

**Comment by camiplata** [comment]
International Cichorieae Network (ICN): Cichorieae Portal has been changed to cc-by

**Comment by camiplata** [comment]
- Russian Academy of Sciences has agree to change all its checklist to cc-by by its next monthly release on April
 -  TaxRef has updated its license to CC-BY

**Comment by mdoering** [comment]
@camiplata @DianRHR Are the missing licenses above still the current state? We should get in touch with Christoph re Fauna Europaea ...

**Comment by camiplata** [comment]
Yes, they are up to date, nevertheless [ this Google doc](https://docs.google.com/spreadsheets/d/1CU3rR_RC848YnUoe7JpaPfrfRSI3b3-OkVYiAXESjwQ/edit?usp=sharing) has further details.

**Comment by mdoering** [comment]
The latest build has some NC sources from publishers:
>2024-07-12 19:22:43,502 WARN  SectorDao                        License CC_BY_NC of source 9640 from publisher d8d6c888-92e7-4036-9808-fb3812be61e1 is not compatible with license CC_BY of project 3. Do not create any sector for: A preliminary checklist of some insects (Coleoptera, Odonata, Hymenoptera) from Arabuko-Sokoke Forest, Kenya
2024-07-12 19:22:43,507 WARN  SectorDao                        License CC_BY_NC of source 28488 from publisher d8d6c888-92e7-4036-9808-fb3812be61e1 is not compatible with license CC_BY of project 3. Do not create any sector for: A Checklist of the Butterflies of Arabuko-Sokoke Forest, Kenya
2024-07-12 19:22:43,511 WARN  SectorDao                        License CC_BY_NC of source 28545 from publisher d8d6c888-92e7-4036-9808-fb3812be61e1 is not compatible with license CC_BY of project 3. Do not create any sector for: A checklist of plants of Arabuko-Sokoke Forest, Kenya, 2019
2024-07-12 19:22:43,515 WARN  SectorDao                        License CC_BY_NC of source 28597 from publisher d8d6c888-92e7-4036-9808-fb3812be61e1 is not compatible with license CC_BY of project 3. Do not create any sector for: A checklist of the reptiles and amphibians of Arabuko-Sokoke Forest, Kenya, 2019
2024-07-12 19:22:43,522 WARN  SectorDao                        License CC_BY_NC of source 29647 from publisher d8d6c888-92e7-4036-9808-fb3812be61e1 is not compatible with license CC_BY of project 3. Do not create any sector for: A Checklist of the Birds of Arabuko-Sokoke Forest, Kenya

**Comment by camiplata** [comment]
Weird that the release from last week worked fine,  the publisher  related to these datasets was added already two weeks ago.

<img width="936" alt="Captura de pantalla 2024-07-14 a la(s) 10 30 11 p m" src="https://github.com/user-attachments/assets/38d360f3-bb89-4259-b6ec-68c8df7e636c">

It looks like if last time the cc by-nc dataset where ignored and the build continued with out apparent issues

<img width="919" alt="Captura de pantalla 2024-07-14 a la(s) 10 39 17 p m" src="https://github.com/user-attachments/assets/0aa6187f-f205-4619-bd9b-09f3d704379a">

**Comment by mdoering** [comment]
... the same thing happened this time. But maybe we can convince the publisher to use a different license?

**Comment by camiplata** [comment]
This will be a continuous task, most of the key sources have now an adequate license
